tests/bench/bench_marshal_test.go

```diff
@@ -1,36 +1,55 @@
 package bench
 
 import (
-	"encoding/json"
-	"math/rand"
+	"os"
 	"testing"
 	"time"
 
 	"github.com/gocql/gocql"
 )
 
-func generateRandomBinaryData(size int) []byte {
-	data := make([]byte, size)
-	rand.Read(data)
-	return data
+type serializationExample struct {
+	Name     string
+	Data     []byte
+	FilePath string
 }
 
-func generateRandomJSON(size int) string {
-	data := make(map[string]interface{})
-	for i := 0; i < size/10; i++ {
-		data[generateRandomString(5)] = generateRandomString(10)
-	}
-	jsonData, _ := json.Marshal(data)
-	return string(jsonData)
+var serializationBinaryExamples = []serializationExample{
+	{
+		Name:     "Small",
+		FilePath: "../../testdata/serialization/small_binary_data.bin",
+	},
+	{
+		Name:     "Medium",
+		FilePath: "../../testdata/serialization/medium_binary_data.bin",
+	},
+	{
+		Name:     "Big",
+		FilePath: "../../testdata/serialization/big_binary_data.bin",
```

Comment:
it is not ok to save big files in git without good reason.
Let's generate json data by [gofakeit](https://github.com/brianvoe/gofakeit/): here is [example](https://github.com/brianvoe/gofakeit/blob/master/json_test.go)
Don't roget to set `Seed` to make it generate same data all the times.

For binary use following construction:
```
	rnd := rand.New(rand.NewSource(100))
	randomBuffer := make([]byte, size)
	_, err := io.ReadAtLeast(rnd, randomBuffer, size)
```

Reply:
should I have smaller exampes saved in files and larger ones generated using the tool that you mentioned or should I sue these tools for every size data?

Reply:
No need to, if we have logic to generate data, why not to generate all of it.